app/views/system_admin/dttp_users/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render PageTitle::View.new(text: @provider.name, has_errors: @provider.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: t("back"),
+    href: providers_path,
+    ) %>
+<% end %>
+
+<h1 class="govuk-heading-l"><%= @provider.name %></h1>
+
+<p> <%= govuk_link_to "Edit this provider", edit_provider_path(@provider), { class: "govuk-link govuk-link--no-visited-state" } %> </p>
+
+<%= render TabNavigation::View.new(items: [
+  { name: "Registered users", url: provider_path(@provider) },
+  { name: "Dttp users", url: provider_dttp_users_path(@provider) },
+]) %>
```

Comment:
This should be a partial as its shared by the `show.html.erb` page and any future pages.